### DISCUSSION TITLE
Please help me to hide the page-separator compontent

### DISCUSSION THREAD

**Discussion by killersolyom** [rejetto/hfs#767] [Q&A] [answered]
Earlier i asked for help to hide the pagination view from the bottom of the page. This is displayed when more than 100 items are displayed. This was the solution: #paging { display: none }

However i recently discovered a bug related to this. Every 100th item will have an extra margin top which ruins the collection view.

![padding](https://github.com/user-attachments/assets/9bca564d-b1ec-4646-ba46-3c7a9f6809ea)

I know what needs to be modified and i can do it from inspect elements 😆 but unfortunately not from custom style.
![component](https://github.com/user-attachments/assets/2b5f973d-70db-48ac-93e4-b926cad500b3)

I tried these:
```
#li.page-separator { display: none }

#ul.dir li.page-separator { display: none }

.ul.dir li.page-separator {
    margin-top: 2.7em;
}
```

**Comment by rejetto** [comment, marked as answer]
hey!
yes, it's a bug, so it must be fixed in hfs, not with custom style.
i will include this in beta6, maybe published tomorrow, but you can right-click on check-for-updates to install this update
https://github.com/user-attachments/files/17294812/hfs-windows-x64-0.54.0-beta5.3.zip

your attempts probably don't work because have less priority then other style.
in this cases you can use the `!important` flag, or make the rule heavier, like this
.tiles-mode ul.dir li.page-separator {     margin-top: 0; }
but i didn't test this

**Comment by killersolyom** [comment]
Thanks for the info and help. I thought i could solve it from custome style. Regarding the attached beta hfs, it works fine.

![image](https://github.com/user-attachments/assets/ae19080e-b245-4c17-add2-a083531c123b)

**Comment by rejetto** [comment]
surely you could, but then you would better remember to remove your hack/fix after the bug was fixed by me, as it could potentially become a problem one day.
and thanks for reporting!
